Sofa/framework/Type/src/sofa/type/Quat.h

```diff
@@ -43,7 +46,16 @@ class SOFA_TYPE_API Quat
     typedef Real value_type;
     typedef sofa::Size Size;
 
-    Quat();
+    constexpr Quat()
+    {
+        this->clear();
+    }
```

Comment:
I dont think the constexpr will have any effect here, Quat::clear() is not constexpr itself